test/bin/ci_phase_iso_build.sh

```diff
@@ -78,22 +80,28 @@ update_build_cache() {
 run_image_build() {
     local -r with_cached_data=$1
 
+    mirror_registry=""
+    if [ "${MIRROR_REGISTRY}" ]; then
+        mirror_registry="-m"
+    fi
+
     if [ -v CI_JOB_NAME ] ; then
+        # CI always mirrors the registry
         # Conditional per-layer builds when running in CI
         if ${with_cached_data} ; then
-            $(dry_run) bash -x ./bin/build_images.sh -d -l ./image-blueprints/layer1-base
+            $(dry_run) bash -x ./bin/build_images.sh -m -d -l ./image-blueprints/layer1-base
         else
-            $(dry_run) bash -x ./bin/build_images.sh    -l ./image-blueprints/layer1-base
+            $(dry_run) bash -x ./bin/build_images.sh -m   -l ./image-blueprints/layer1-base
         fi
 
-        $(dry_run) bash -x ./bin/build_images.sh -l ./image-blueprints/layer2-presubmit
+        $(dry_run) bash -x ./bin/build_images.sh -m -l ./image-blueprints/layer2-presubmit
 
         if [[ "${CI_JOB_NAME}" =~ .*periodic.* ]]; then
-            $(dry_run) bash -x ./bin/build_images.sh -l ./image-blueprints/layer3-periodic
+            $(dry_run) bash -x ./bin/build_images.sh -m -l ./image-blueprints/layer3-periodic
         fi
     else
         # Fall back to full build when not running in CI
-        $(dry_run) bash -x ./bin/build_images.sh
+        $(dry_run) bash -x ./bin/build_images.sh "${mirror_registry}"
```

Comment:
Why do we need this? If the intention is to allow enabling mirror in devenv, it can be done by `export CI_JOB_NAME=qqq`, right?

Reply:
Yes, but I find it a bit confusing not to have a dedicated option. Plus, the CI_JOB_NAME makes you follow a different path.